<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="switchPlayer2Button">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>180</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch Player 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="switchPlayer1Button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Switch Player 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="player2TimeButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>set Player 2 Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="player1TimeButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>set Player 1 Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startGameButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="player1TimeBar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="player2TimeBar">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="player1TimeLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 1 Time :</string>
    </property>
   </widget>
   <widget class="QLabel" name="player2TimeLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 2 Time :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
